assignment1/hello_dj_tmc.py

```diff
@@ -0,0 +1 @@
+print("Hello, World!")
```

Comment:
you need a if __name__ == "__main__" statement

Reply:
Thanks, will do!